web/themes/custom/parc/js/script.js

```diff
@@ -67,4 +67,15 @@
       }, false);
     }
   };
+
+  Drupal.behaviors.thematicAreas = {
+    attach: function (context, settings) {
+      var section_id;
```

Comment:
Once we implement this dynamic part, it's clear that content editors will not be able to change it. It will be for the developers to make any change in this part.